plugin/evm/block.go

```diff
@@ -101,15 +106,72 @@ func (b *Block) syntacticVerify() error {
 }
 
 // Verify implements the snowman.Block interface
+// Since VerifyWithContext is implemented, we never expect this to be called.
```

Comment:
I think we should either remove or clarify this comment since `Verify` is still expected to be called in certain cases.

Reply:
Good call. Removed.